json-schemas/src/account-stats.json

```diff
@@ -1097,6 +1094,36 @@
           "inclusiveMinimum": 0,
           "description": "Total number of messages on channels where delta generation was attempted but where a delta was not successfully generated, so Ably just broadcasted the original, see https://ably.com/docs/channels/options/deltas."
         },
+        "messages.processed.batching.simple.inboundMessages": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of messages going through the batching pipeline with the 'simple' batching policy."
+        },
+        "messages.processed.batching.simple.generatedBatches": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of batches generated by the 'simple' batching policy."
+        },
+        "messages.processed.batching.simple.discardedMessages": {
```

Comment:
given this should always be 0, do we need to include it? in other places we haven't bothered to include entries for stats which will never be non-zero